python/ray/rllib/a3c/a3c.py

```diff
@@ -38,8 +38,8 @@
     "vf_loss_coeff": 0.5,
     # Entropy coefficient
     "entropy_coeff": -0.01,
-    # Preprocessing for environment
-    "preprocessing": {
+    # Model and preprocessor options
+    "model": {
```

Comment:
why do we want to share this field?

Reply:
This was just for consistency with the other algorithms. I don't have a strong opinion either way but we should do it consistently across algorithms.